libraries/standard/http/include/http_client.h

```diff
@@ -84,7 +84,7 @@ typedef struct HTTPNetworkContext HTTPNetworkContext_t;
 /**
  * @brief Transport interface for sending data over the network.
  *
- * If the number of bytes written returned is less than bytesToWrite, then
+ * If the number of bytes written returned is not equal to bytesToWrite, then
```

Comment:
(Nit) `@p bytesToWrite`

Reply:
I didn't know about that doxygen I will add it everywhere thanks